### PR TITLE
Update zap stanza for iStat Menus

### PR DESCRIPTION
**Pull request by ondrejfuhrer**
Updated zap stanza for the latest version.

There are still some remains, that I'm not sure, how to remove, those are located in `/private/var/folders/` folder under a hash, which I'm not sure if is changing or not, can someone take a look?

### CODE REVIEW

File: Casks/istat-menus.rb [hunk 1]
```diff
@@ -28,13 +29,21 @@
...
                '~/Library/Caches/com.bjango.iStatMenusAgent',
+               '~/Library/Caches/iStat Menus',
+               '~/Library/Containers/com.bjango.istatmenus.iStat-Menus-Widget',
+               '~/Library/Cookies/com.bjango.istatmenus.binarycookies',
                '~/Library/Logs/iStat Menus',
                '~/Library/Preferences/com.bjango.istatmenus.plist',
                "~/Library/Preferences/com.bjango.istatmenus#{version.major}.extras.plist",
-               '~/Library/Preferences/com.bjango.istatmenusstatus.plist',
+               '~/Library/Preferences/com.bjango.istatmenus.status.plist',
+               '/Library/PrivilegedHelperTools/com.bjango.istatmenus.installerhelper',
```
Comment (commitay) [inline Casks/istat-menus.rb]: `PrivilegedHelperTools` should be moved to `uninstall delete`

File: Casks/istat-menus.rb [hunk 2]
```diff
@@ -13,10 +13,11 @@
...
-                         'com.bjango.istatmenusagent',
-                         'com.bjango.istatmenusdaemon',
+                         'com.bjango.istatmenus.agent',
+                         'com.bjango.istatmenus.daemon',
                          'com.bjango.istatmenusnotifications',
-                         'com.bjango.istatmenusstatus',
+                         'com.bjango.istatmenus.status',
+                         'com.bjango.istatmenus.installerhelper',
                        ],
             signal:    [
```
Comment (commitay) [inline Casks/istat-menus.rb]: Does the `signal` stanza need to be updated as well?

Comment (ondrejfuhrer) [reply Casks/istat-menus.rb]: I will double check but seems there is nothing running after the zap is done.

Comment (ondrejfuhrer) [reply Casks/istat-menus.rb]: Just a side note (didn't have time to go through it completely and test it), there are a lot of "old" things to remove, that are not valid anymore. I will clean it up and create a new PR. Thanks for the review!

**Pull request merged**